diff --git a/data/xyz.tytanium.DoorKnocker.appdata.xml.in.in b/data/xyz.tytanium.DoorKnocker.appdata.xml.in.in
index b49acca..b4c0295 100644
--- a/data/xyz.tytanium.DoorKnocker.appdata.xml.in.in
+++ b/data/xyz.tytanium.DoorKnocker.appdata.xml.in.in
@@ -24,6 +24,7 @@ SPDX-License-Identifier: CC0-1.0
   <developer id="tytan652">
     <name>tytan652</name>
   </developer>
+  <developer_name>tytan652</developer_name>
   <translation type="gettext">door-knocker</translation>
 
   <screenshots>
